lerobot/common/robot_devices/motors/dynamixel.py

```diff
@@ -154,6 +143,8 @@
 NUM_READ_RETRY = 10
 NUM_WRITE_RETRY = 10
 
+COMM_SUCCESS = 0
```

Comment:
Nit: I don't love that `COMM_SUCCESS` is hardcoded here instead of being imported from dynamixel_sdk....

Reply:
Good catch! Removed this line. Added dynamic import everywhere
```python
if self.mock:
    from tests.mock_dynamixel import GroupSyncWrite, COMM_SUCCESS
else:
    from dynamixel_sdk import GroupSyncWrite, COMM_SUCCESS
    ```